packages/component-library/src/components/form/mt-switch/mt-switch.vue

```diff
@@ -30,7 +30,6 @@
           :is-inheritance-field="isInheritanceField"
           :is-inherited="isInherited"
           @inheritance-restore="onInheritanceRestore($event)"
```

Comment:
What about the restore event?

Reply:
Restoring the inheritance didn't lead to issues. I just want to avoid changes here which can break other stuff. So these are only the necessary fixes to make it work.

But after the major we should look at all foundings and try to improve it